x/project/internal/keeper/keeper.go

```diff
@@ -214,13 +217,13 @@ func (k Keeper) ClaimExists(ctx sdk.Context, projectDid did.Did, claimId string)
 	return store.Has(types.GetClaimKey(projectDid, claimId))
 }
 
-func (k Keeper) GetClaim(ctx sdk.Context, projectDid did.Did, claimId string) (types.Claim, sdk.Error) {
+func (k Keeper) GetClaim(ctx sdk.Context, projectDid did.Did, claimId string) (types.Claim, error) {
 	store := ctx.KVStore(k.storeKey)
 	key := types.GetClaimKey(projectDid, claimId)
 
 	bz := store.Get(key)
 	if bz == nil {
-		return types.Claim{}, did.ErrorInvalidDid(types.DefaultCodespace, "claim not found")
+		return types.Claim{}, fmt.Errorf("claim not found")
```

Comment:
Why not use `sdkerrors.Wrap(did.ErrInvalidDid, "claim not found")`?

Reply:
Claims are not identified by DIDs and we do not have a custom error type for invalid claims.

Reply:
Will leave it as is